src/NuGet.Core/NuGet.Packaging/Signing/SignatureIssue.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Common;
+
+namespace NuGet.Packaging.Signing
+{
+    public class SignatureIssue : IEquatable<SignatureIssue>
+    {
+        public bool Fatal { get; }
+
+        public string Message { get; }
+
+        public NuGetLogCode Code { get; }
+
+        private SignatureIssue(bool fatal, NuGetLogCode code, string message)
+        {
+            Fatal = fatal;
+            Code = code;
+            Message = message;
+        }
+
+        public static SignatureIssue InvalidInputError(string message)
```

Comment:
Add a verb prefix maybe

Reply:
@emgarten I was following the same standard as [CompatibilityIssue](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Commands/RestoreCommand/CompatibilityIssue.cs#L65). Should I prepend a `Create` to all of these static methods?